install-selenium

From: Bryan Larsen <bryan@larsen.st>

# Integration Testing

It's not a real application without tests.  We're going to use [Selenium](http://seleniumhq.org/) to do some integration testing.

We've forked selenium-on-rails to add a couple of helper functions.  Install our fork using

    $ git submodule add git://github.com/bryanlarsen/selenium-on-rails.git vendor/plugins/selenium-on-rails
---

 .gitmodules                      |    3 +++
 vendor/plugins/selenium-on-rails |    1 +
 2 files changed, 4 insertions(+), 0 deletions(-)
 create mode 160000 vendor/plugins/selenium-on-rails


diff --git a/.gitmodules b/.gitmodules
index 686818e..cdaf2ed 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -4,3 +4,6 @@
 [submodule "vendor/plugins/acts_as_list"]
 	path = vendor/plugins/acts_as_list
 	url = git://github.com/rails/acts_as_list.git
+[submodule "vendor/plugins/selenium-on-rails"]
+	path = vendor/plugins/selenium-on-rails
+	url = git://github.com/bryanlarsen/selenium-on-rails.git
diff --git a/vendor/plugins/selenium-on-rails b/vendor/plugins/selenium-on-rails
new file mode 160000
index 0000000..ff6b05a
--- /dev/null
+++ b/vendor/plugins/selenium-on-rails
@@ -0,0 +1 @@
+Subproject commit ff6b05ad1f32186a4b4b0220a9a662ec5ca5a91d
